napari/layers/tracks/tracks.py

```diff
@@ -381,8 +382,11 @@ def properties(self, properties: Dict[str, np.ndarray]):
         if self._color_by not in [*properties.keys(), 'track_id']:
             warn(
                 (
-                    f"Previous color_by key {self._color_by!r} not present in"
-                    " new properties. Falling back to track_id"
+                    trans._(
+                        "Previous color_by key {key} not present in new properties. Falling back to track_id",
+                        deferred=True,
+                        key=f"{self._color_by!r}",
+                    )
```

Comment:
See question about !r above if there is a resolution.

Reply:
Yes you are right, I should have left that inside... I am not sure what I was thinking... maybe I was missing my ☕

Reply:
Fixed, added a check and discovered a few others :)